tests/framework/PostProcessors/Validation/test_validation_gate_pcm_Snapshot.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
```

Comment:
The filename is not as same as the one used in tests file, please check it @ShimingYIN 
Traceback (most recent call last):
  File "/home/civet/civet/build_0/raven/raven_framework.py", line 25, in <module>
    sys.exit(main(True))
  File "/home/civet/civet/build_0/raven/ravenframework/Driver.py", line 127, in main
    tree = TS.parse(open(inputFile,'r'))
FileNotFoundError: [Errno 2] No such file or directory: '/home/civet/civet/build_0/raven/tests/framework/PostProcessors/Validation/test_validation_gate_pcm_SnapshotSeth.xml'